index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio | Edwin Gichuhi</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <a href="#">My Logo</a>
```

Comment:
- [ ] it would be better to stick to the Figma design and try. to make your logo the same as the one in the Figma design by removing the underline and make sure the font-weight is working   in order to match it 
-Yours
<img width="87" alt="Screen Shot 2022-09-28 at 17 41 31" src="https://user-images.githubusercontent.com/61790587/192839963-04bc1e24-4c33-47ac-a4db-fe295b0017ca.png">
- Figma
<img width="80" alt="Screen Shot 2022-09-28 at 17 41 37" src="https://user-images.githubusercontent.com/61790587/192840030-a2908f66-a089-465b-bdfb-7050a968ea8e.png">